src/lib/bcmp/bcmp_config.cpp

```diff
@@ -7,17 +8,14 @@ extern "C" {
 #include "packet.h"
 }
 
-static Configuration *USR_CFG;
-static Configuration *SYS_CFG;
```

Comment:
removing all of the configuration.h/.cpp specifics from this file and moving them to `bm_config_wrapper.cpp`. all of the other changes in this file are converting to the enums/structs to the new code style(and using the type definitions from `bm_core/messages.h`)